Recovering missing --color pieces from textui docs

Jean85 opened the pull request:
As found during #319.

I've recovered all the missing pieces in the english file.

I've ported what I could to the translated versions, here is what I've found incomplete or recovered (as you can see from the diff):
 * fr:
  * `--colors` is correctly shown in CLI example, translated explanation still old version
  * `--strict-global-state` absent in CLI example, translated explanation still missing
  * `--strict` was still there (was deprecated and removed), removed from docs
 * ja, pt_br, zh_cn:
  * `--colors` old in CLI example, updated; translated explanation still old version

henriquemoody commented on src/4.8/en/textui.xml:
```diff
@@ -647,6 +647,39 @@ class TestCaseClass extends \PHPUnit_Framework_TestCase
+              </listitem>
+              <listitem>
+                <para>
+                  <literal>always</literal>: always displays colors in the output even when the current terminal doesn't supports colors,
```
There is a typo (my fault) on this and the other 2 topics above:

> current terminal doesn't supports colors

It must be _current terminal doesn't support colors_.

Jean85 replied:
Fixed in next commit (358e15262bfde9bc9401a154b06d3d69d16bb44f)

henriquemoody commented on src/4.8/pt_br/textui.xml:
```diff
@@ -113,7 +113,7 @@ OK (2 tests, 2 assertions)</screen>
 
     <screen><userinput>phpunit --help</userinput>
-<![CDATA[PHPUnit 4.7.0 by Sebastian Bergmann and contributors.
+<![CDATA[PHPUnit 4.8.0 by Sebastian Bergmann and contributors.
```
Nice catch.

Jean85 replied:
I just copy-pasted the full CLI output to "Catch 'Em All!!" :laughing:

Jean85 commented:
[ping] @sebastianbergmann ? Can this be merged?

sebastianbergmann commented:
The same changes need to be made for `5.0` and `5.1`.

Jean85 commented:
Working on it!

Jean85 commented:
@sebastianbergmann done.
I've also removed a lot of wrong paragraphs about `--strict`, that was deprecated and dropped, and that were present in a lot of >=4.8 docs.

Jean85 commented:
@sebastianbergmann I think it's ready to be merged..
